suseconnect-ng.spec

```diff
@@ -16,60 +16,44 @@
 #
 
 
-%global provider_prefix github.com/SUSE/connect-ng
-%global import_path     %{provider_prefix}
-
-# set this to enable hwinfo test in %check
-%bcond_with hwinfo
+%global project github.com/SUSE/connect-ng
 
 Name:           suseconnect-ng
-# the version will get set by the 'set_version' service
-Version:        1.1.0~git2.f42b4b2
+Version:        1.7.0
```

Comment:
The version is still applied by `set_version` above, right?

Reply:
yep. Do you want back the comment? I find it unnecessary since the version supplied here is actually already overwritten by `set_version` at this stage.

Reply:
I don't mind either of the two, was just curious :+1: